charon/tests/ui/type_layout.out

```diff
@@ -0,0 +1,23 @@
+# Final LLBC before serialization:
```

Comment:
The pretty-printed output won't include layout info, to test this add a test in `tests/crate_data.rs`